lib/Config/TranslationUpdater.php

```diff
@@ -77,10 +77,12 @@ private function getTranslationUpdates(array $locales): array {
       $requestBody['plugins'][$this->premiumSlug] = ['version' => $this->premiumVersion];
     }
 
+    // Three seconds, plus one extra second for every 10 plugins.
+    $timeout = 3 + (int)(count($locales) / 10);
```

Comment:
Are 3 seconds enough? Should we keep it 10s or so to be safe?

Reply:
Good point! I used the pattern from the WC plugin. We can use 10 seconds as a minimum.